doc/metric_spec.md

```diff
@@ -102,6 +102,12 @@ The value can vary from 0, 1 , 5 etc to `+Inf`, that  correspond to the infinity
 The number of migration threshold pro node and resource ID set by a pacemaker cluster. 
 Possible values are positive numbers.
 
+### `ha_cluster_pacemaker_resource_configuration_changes`
+
+#### Description
+
+This metric return a unix timestamp which correspond to the C.ansi timestamp updated by pacemaker in cluster.
+It is useful to see if the resource agent configuration in the cluster has changed over a period of time and gets alerts.
```

Comment:
We just call it "resource configuration" rather than "resource agent configuration". A resource is of course managed via an agent.